public/viewer.js

```diff
@@ -7,12 +7,12 @@ var twitch = window.Twitch.ext;
 // base url for our backend
 var backendUrl = 'https://localhost:8081';
 
-function createPollRequest(text) {
+function createPollRequest(text, textA, textB) {
 
     return {
         type: 'POST',
         url: backendUrl + '/poll/create',
-        data: {'text': text},
+        data: {'text': text, 'textA': textA, 'textB': textB},
```

Comment:
fine for exactly two choices - better perhaps would be to have another object inside of here called options... ie....

data: {'text': text, 'options': options}
where options = [...]
an ordered list of options, each option itself having a label text (that the viewer will see), and a value (how it will be represented internally)

if options is a list, we can iterate over it and do things like create the exact number of buttons we need to display for a variable number of options